internal/client/resource_group.go

```diff
@@ -0,0 +1,132 @@
+package client
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+const resourceGroupBasePath = "/api/resource_groups"
+
+type ResourceGroup struct {
+	ID          int64   `json:"id"`
+	Name        string  `json:"name"`
+	Description *string `json:"description"`
```

Comment:
The previous and current definitions seem to be slightly different. Is everything OK?
Have you verified that the data mart is working?

Reply:
The difference is a mistake. I'll fix later, thx!

```suggestion
	Description string `json:"description"`
```